dts/arm/renesas/ra/ra8/ra8x1.dtsi

```diff
@@ -22,12 +22,26 @@
 			reg = <0>;
 			#address-cells = <1>;
 			#size-cells = <1>;
+			cpu-power-states = <&stop0 &stop1>;
 
 			mpu: mpu@e000ed90 {
 				compatible = "arm,armv8.1m-mpu";
 				reg = <0xe000ed90 0x40>;
 			};
 		};
+
+		power-states {
+			stop0: state0 {
+					compatible = "zephyr,power-state";
+					power-state-name = "runtime-idle";
+					min-residency-us = <300>;
+				};
+			stop1: state1 {
+					compatible = "zephyr,power-state";
+					power-state-name = "standby";
+					min-residency-us = <50000>;
+				};
```

Comment:
alignment wrong on end brackets, indent wrong on middle lines

Reply:
It's fixed